app/src/main/kotlin/com/wire/android/ui/home/newconversation/search/InternalContactSearchResultItem.kt

```diff
@@ -110,11 +114,17 @@ fun ExternalContactSearchResultItem(
             )
         },
         actions = {
-            if (!isConnectedOrPending) AddContactButton({ })
+            when (connectionStatus) {
+                ConnectionStatus.NotConnected -> AddContactButton(onAddContactClicked)
+                ConnectionStatus.Pending -> Box(modifier = Modifier.padding(horizontal = 12.dp)) { ConnectPendingRequestBadge() }
+                ConnectionStatus.Sent -> Box(modifier = Modifier.padding(horizontal = 12.dp)) { ConnectRequestBadge() }
+                ConnectionStatus.Connected,
+                ConnectionStatus.Unknown -> {
+                }
```

Comment:
What do you think of adding here something like a log? so we can be aware that this is happening in prod (but maybe it shouldn't)

Reply:
Sure, will add logs 💪